base/PerfOptions.php

```diff
@@ -235,7 +235,7 @@ public function __construct(Vector<string> $argv) {
         $this->tcprint = $tcprint;
       } else if ($isFacebook) {
         $this->tcprint =
-          $fbcode . '/_bin/hphp/facebook/tools/tc-print/tc-print';
+          $fbcode . '/_bin/hphp/tools/tc-print/tc-print';
```

Comment:
We could infer this from $this->hhvm without gating by isFacebook now, couldn't we?